subprojects/core/src/main/java/org/gradle/api/tasks/JavaExec.java

```diff
@@ -337,9 +372,16 @@ public FileCollection getClasspath() {
      */
     public JavaExec copyTo(JavaForkOptions options) {
```

Comment:
Should we deprecate this guy?

Reply:
And move the method from `JavaForkOptions` to `JavaForkOptionsInternal`?

Reply:
Eventually, yes.